server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -841,7 +841,6 @@ public void run() {
         try (TabletsMetadata tabletsMetadata =
```

Comment:
I believe this will absolutely fix the problem.  I am wondering if changing the "watchCriticalFixedDelay" on line 826 can simply be changed to "watchNonCriticalScheduledTask" ?